### PR TITLE
Remove nebula-release plugin

### PR DESCRIPTION
**Pull request by Joaquimmnetto**
## Description
Removed the nebula-release plugin, used for version management. Versions are now managed by the added [atlas-version](https://github.com/wooga/atlas-version) plugin.

## Changes
* ![ADD] atlas-version plugin
* ![REMOVE] nebula-release plugin

[NEW]:https://resources.atlas.wooga.com/icons/icon_new.svg "New"
[ADD]:https://resources.atlas.wooga.com/icons/icon_add.svg "Add"
[IMPROVE]:https://resources.atlas.wooga.com/icons/icon_improve.svg "IMPROVE"
[CHANGE]:https://atlas-resources.wooga.com/icons/icon_change.svg "Change"
[FIX]:https://atlas-resources.wooga.com/icons/icon_fix.svg "Fix"
[UPDATE]:https://atlas-resources.wooga.com/icons/icon_update.svg "Update"

[BREAK]:https://atlas-resources.wooga.com/icons/icon_break.svg "Remove"
[REMOVE]:https://resources.atlas.wooga.com/icons/icon_remove.svg "Remove"
[IOS]:https://atlas-resources.wooga.com/icons/icon_iOS.svg "iOS"
[ANDROID]:https://atlas-resources.wooga.com/icons/icon_android.svg "Android"
[WEBGL]:https://atlas-resources.wooga.com/icons/icon_webGL.svg "WebGL"
[UNITY]:https://atlas-resources.wooga.com/icons/icon_unity.svg "Unity"
[GRADLE]:https://atlas-resources.wooga.com/icons/icon_gradle.svg "Gradle"
[LINUX]:https://atlas-resources.wooga.com/icons/icon_linux.svg "Linux"

### CODE REVIEW

File: src/integrationTest/groovy/wooga/gradle/plugins/PluginsPluginIntegrationSpec.groovy [hunk 1]
```diff
@@ -72,7 +98,7 @@ class PluginsPluginIntegrationSpec extends IntegrationSpec {
     }
 
     @Unroll
-    def "task :#lifecycleTask runs :#taskToRun"() {
+    def "task :#lifecycleTask runs dependencies"() {
```
Comment (Larusso) [inline src/integrationTest/groovy/wooga/gradle/plugins/PluginsPluginIntegrationSpec.groovy:101]: why change that. I think the test name becomes clearer

File: src/integrationTest/groovy/wooga/gradle/plugins/PluginsPluginIntegrationSpec.groovy [hunk 2]
```diff
@@ -16,27 +16,49 @@
...
 import org.junit.contrib.java.lang.system.EnvironmentVariables
 import spock.lang.IgnoreIf
+import spock.lang.Shared
 import spock.lang.Unroll
 
 class PluginsPluginIntegrationSpec extends IntegrationSpec {
 
+    @Shared
+    @GithubRepository(
+            usernameEnv = "ATLAS_GITHUB_INTEGRATION_USER",
```
Comment (Larusso) [inline src/integrationTest/groovy/wooga/gradle/plugins/PluginsPluginIntegrationSpec.groovy:34]: I guess you need the same changes to the Jenkins file as you had in the other PR

File: src/integrationTest/groovy/wooga/gradle/plugins/PluginsPluginIntegrationSpec.groovy (hunk 1 above)
Comment (Joaquimmnetto) [reply src/integrationTest/groovy/wooga/gradle/plugins/PluginsPluginIntegrationSpec.groovy:101]: When I wrote this I wasnt aware on how to make the test title work with arrays. Now I know how to do it. I'll do the change.

Comment (Joaquimmnetto) [reply src/integrationTest/groovy/wooga/gradle/plugins/PluginsPluginIntegrationSpec.groovy]: Done.

**Review by Larusso — APPROVED** [review]
Was this PR rebased? There are changes in the change list renderer I can't really understand

**Pull request merged**